public/src/main/java/com/revenuecat/purchases/models/PurchaseInfo.kt

```diff
@@ -0,0 +1,8 @@
+package com.revenuecat.purchases.models
+
+import com.revenuecat.purchases.ProductType
+
+interface PurchaseInfo {
```

Comment:
Additionally, I wonder if this name could be confusing... Since this is meant to start the purchase process, maybe `MakePurchaseInfo` or `MakePurchaseData`?

Reply:
I was also thinking it could use a name change 😛  `MakePurchaseInfo` sounds like a function name but I do think it should be something close to that 🤔